storybook/stories/_support/DiscreetModeNotificationKnob.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { useLayoutEffect } from 'react';
+import { observer } from 'mobx-react';
+import { boolean } from '@storybook/addon-knobs';
+import { useDiscreetModeFeature } from '../../../source/renderer/app/features';
+
+export const DiscreetModeNotificationKnob = observer(() => {
+  const feature = useDiscreetModeFeature();
+  const tooltipKnob = boolean('Discreet mode tooltip', false, 'Dicreet Mode');
+  const notificationKnob = boolean(
+    'Discreet mode notification',
```

Comment:
Are those strings not localized ?

Reply:
I don't thing is necessary. This is just for storybook controls.